beacon-chain/forkchoice/interfaces.go

```diff
@@ -16,13 +16,22 @@ type BalancesByRooter func(context.Context, [32]byte) ([]uint64, error)
 
 // ForkChoicer represents the full fork choice interface composed of all the sub-interfaces.
 type ForkChoicer interface {
+	ForkChoiceLocker     // to lock forkchoice
 	HeadRetriever        // to compute head.
 	BlockProcessor       // to track new block for fork choice.
 	AttestationProcessor // to track new attestation for fork choice.
 	Getter               // to retrieve fork choice information.
 	Setter               // to set fork choice information.
 }
 
+// ForkChoiceLocker defines an interface for methods that aquire and release the forkchoice locks
+type ForkChoiceLocker interface {
```

Comment:
`forkchoice.ForkChoiceLocker` is verbose, how about just `forkchoice.Locker` ?

```suggestion
type Locker interface {
```